src/Copy/BrandCopy.elm

```diff
@@ -81,7 +81,7 @@ brandCopy key =
             CallToAction
                 { action = DesktopInteractive
                 , category = "email"
-                , icon = "✨"
+                , icon = "email"
```

Comment:
Do you mean envelop? Or you have included 2 alternatives?

Reply:
2 alternatives. I will remove the one unused.